### ISSUE TITLE
[SPIKE] Preliminary work for SiteImprove test installations.

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#16207]
## Description

Followup from #15923 .

Basically, we need to install the SiteImprove module and determine whether it has any issues preventing it from being deployed or tested in Tugboat, Test-Staging, or local environments.

## Tasks
- Install and enable module in a feature branch.
- Determine whether any issues have occurred already, and if so, fix them.
- Open a PR in draft to have a reviewable Tugboat preview.
- Push branch to `va.gov-cms-test` and deploy to test.staging.
- Determine whether any issues have occurred already, and if so, fix them.

## Acceptance Criteria

- [ ] Module has been installed.
- [ ] Scaffolding has been added to bring in API key, etc from appropriate source in given environment.
- [ ] Any blocking issues have been resolved or inability to resolve them has been reported to Berni/Marcia/Erika.
- [ ] Followup issue(s) opened for December 4 runbook/communications if no blocking issues found.

**Comment by maortiz-27-80** [comment]
@ariperez - are you able to point this ticket, assuming that we have the token?  we're still waiting on VHA-DM team for the token and will provide that asap.  cc: @ndouglas  @BerniXiongA6  @JunTaoLuo

**Comment by maortiz-27-80** [comment]
@ariperez - Before you get too into the weeds of this ticket, at your earliest convenience, can you confirm the following so we can follow up with VHA-DM and Siteimprove if we need additional assistance:

1. Siteimprove token (for the Siteimprove plugin) works correctly (if token field is empty)
2. Siteimprove API username and API key work correctly for setting up the prepublish check

cc: @BerniXiongA6 @ndouglas @JunTaoLuo @EWashb

**Comment by maortiz-27-80** [comment]
@EWashb -  The team is stuck on getting the API username and API keys to configure the plugin on http://test.prod.cms.va.gov/admin/config/system/siteimprove

With Laura's existing Siteimprove credentials (one for Va.gov and one for Civic Actions), we were unable to generate an API username and API key with [the instructions that were included here](https://help.siteimprove.com/support/solutions/articles/80000448206-how-to-connect-to-the-siteimprove-api#:~:text=API%20Reference-,To%20set%20up%20a%20connection%20to%20the%20API,-Go%20to%20Integrations).  

siteimprove_plugin_01 - plugin displays (floating) 
<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK1pUQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--dcd451b8b3e82e5ac637acb173bd9a94e5750aef/siteimprove_plugin_01.png" alt="siteimprove_plugin_01.png" />

siteimprove_plugin_02 - plugin displays unavailable message when clicked on
<img width="1785" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK2RUQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--693202d7e2e470ea7f6842a92ce18a63d2ab1678/siteimprove_plugin_02.png" alt="siteimprove_plugin_02.png" />

siteimprove_config - config page where we are unable to generate and fill in: (1) API username and (2) API key
<img width="1323" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK2hUQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--16d0b256bf2f97b0c336fc9361a1ab17b9776ddb/siteimprove_config.png" alt="siteimprove_config.png" />

cc: @BerniXiongA6 @ndouglas @ariperez @laflannery @JunTaoLuo

**Comment by maortiz-27-80** [comment]
@ariperez  - VHA-DM has provided username and API key.  let me know if this works:

User Name: Laura Flannery

Key: 5963c49ba007c022f19f029f3b789397

cc: @BerniXiongA6 @EWashb @ndouglas @laflannery  @JunTaoLuo

**Comment by ariperez** [comment]
@maortiz-27-80 , we tried multiple variations of user name for Laura, still no luck.

<img width="1390" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMzlZQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--80b6456ebc822078a51d47806fe355252435ff33/image.png" alt="image.png" />

also, noting that in the SI ui, Laura still cannot access API config functionality, which seems like it should be possible in order for her to manage her API keys...<img width="1574" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNEZZQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--403cc2b8796a6170cf208fd7b1dffc3afedc6444/image.png" alt="image.png" />

https://www.drupal.org/node/3180424

**Comment by maortiz-27-80** [comment]
Per slack convo here, 12/14, this ticket is complete and marking as Closed.  Separate ticket, #10275 will be utilized for implementing the API username and API key and initiating deep linking with SI. cc: @ariperez @BerniXiongA6 @ndouglas @JunTaoLuo @laflannery
